microcosm_pubsub/tests/fixtures.py

```diff
@@ -4,67 +4,64 @@
 """
 from marshmallow import fields, Schema
 
+from microcosm.api import binding
 from microcosm_pubsub.codecs import PubSubMessageSchema
 from microcosm_pubsub.conventions import created
 from microcosm_pubsub.daemon import ConsumerDaemon
 from microcosm_pubsub.decorators import handles, schema
 from microcosm_pubsub.errors import SkipMessage
 
 
-FOO_MEDIA_TYPE = "application/vnd.globality.pubsub.foo"
-FOO_QUEUE_URL = "foo-queue-url"
-FOO_TOPIC = "foo-topic"
-MESSAGE_ID = "message-id"
-RECEIPT_HANDLE = "receipt-handle"
-
-
-class ExampleDaemon(ConsumerDaemon):
-
-    @property
-    def name(self):
-        return "example"
-
-
-class Foo(object):
-    pass
-
-
 @schema
-class FooSchema(PubSubMessageSchema):
+class DerivedSchema(PubSubMessageSchema):
```

Comment:
I tried to get rid of `foo` and `bar` and be more explicit with my naming. This avoids some spurious test cases where we have duplicate names.